d2graph/d2graph.go

```diff
@@ -607,6 +607,9 @@ type Edge struct {
 	MinWidth  int `json:"minWidth"`
 	MinHeight int `json:"minHeight"`
 
+	FromTableRowIndex int `json:"fromTableRowIndex"`
```

Comment:
let's make these pointers and omit empty, since they won't exist most of the time